_data/nav.yml

```diff
@@ -415,6 +415,7 @@
     - url: "#paste_convert_word_fake_lists"
     - url: "#paste_remove_styles_if_webkit"
     - url: "#smart_paste"
+    - url: "#image_file_types"
```

Comment:
These are largely kept in alphabetical order. So should this be before all the paste settings?

Reply:
I only did this due to its direct relationship with `smart_paste` above, but happy to move it

Reply:
I'm ok with breaking the convention for this case if it is related to the `smart_paste` option.